airflow_spell/operators/spell_run.py

```diff
@@ -85,29 +86,29 @@ def execute(self, context: typing.Dict):
         self.monitor_run(context)
         return "Spell run execute completed."
 
-    def submit_run(self, context: typing.Dict):  # pylint: disable=unused-argument
+    def submit_run(self, context: Dict):  # pylint: disable=unused-argument
         self.log.info("Running Spell run")
 
         try:
             run = self.client.runs.new(**self.kwargs)
-            self.run_id = run.id
+            self.spell_run_id = run.id
 
-            self.log.info(f"Spell run ({self.run_id}) started: {run}")
+            self.log.info(f"Spell run (spell_run_id: {self.spell_run_id}) started: {run}")
```

Comment:
Perhaps not to use F strings in logging but %s logging formatting option `"Spell run %s", run"`. Same on few places below as well

Reply:
👍  Will do